limitador/src/storage/redis/counters_cache.rs

```diff
@@ -141,10 +143,11 @@ impl Batcher {
             notifier: Default::default(),
             interval: period,
             priority_flush: AtomicBool::new(false),
+            limiter: Semaphore::new(1000),
```

Comment:
we may want to make this configurable later.  Is there a better default we can use for now?

Reply:
Or we'd need to monitor memory usage and do it all conditional on that... Not sure I want to start going down that rabbit hole tho 🙈

Reply:
Maybe we could, for now, use the same [`max_cached_counters` value](https://github.com/Kuadrant/limitador/blob/65d34f15a35240c9ffdf2a0d0609b68b74dde45e/limitador/src/storage/redis/counters_cache.rs#L379-L380). Gives the user some control, while avoiding leaking too many "implementation details" just quiet yet, wdyt?

Reply:
Sounds good.  Updated.